docs/topics/modules/cleaning_up_oso_account.adoc

```diff
@@ -1,16 +1,24 @@
 [id="cleaning_up_oso_account"]
-= Cleaning up your OpenShift Online Account
+= Cleaning up your OpenShift Online account
 
-{osio} generates artifacts within five projects (Kubernetes namespaces) in OpenShift Online. See <<viewing_projects_oso>> for details. After you create the Vert.X and Spring Boot HTTP quickstart projects, the limited resources available in the OpenShift Online Free Tier are nearly exhausted. You can now reset your {osio} and Openshift Online account resources to start afresh:
+{osio} generates artifacts within five projects (Kubernetes namespaces) in OpenShift Online. See <<viewing_projects_oso>> for details. This section describes how to reset your OpenShift Online account, which can serve two purposes:
+
+Freeing resources used by tutorials:: After you create the <<hello_world_developers,Vert.X>> and <<spring_boot_quickstart_tutorial,Spring Boot HTTP>> quickstart projects, the limited resources available in the OpenShift Online Free Tier are nearly exhausted. You can reset your {osio} and Openshift Online account resources to start afresh.
+
+General troubleshooting:: If your OpenShift.io environment is in a state where you find it difficult or impossible to move forward, you can reset your environment to start afresh.
+
+
+[discrete]
+== Procedure
 
 . In your `Profile` page, click btn:[Update Profile].
```

Comment:
The first step needs to be preceded by these instructions, they were there in the 6th Chpt:
. On your OpenShift.io home page, click your name on the top right corner of the screen.

. In the drop-down options, click Profile.
. In your profile page, click...